bundle/manifests/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -128,7 +128,7 @@ metadata:
       "llamastackoperators.components.platform.opendatahub.io"]'
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v4
     repository: https://github.com/opendatahub-io/opendatahub-operator
-  name: opendatahub-operator.v3.0.0
+  name: opendatahub-operator.v2.33.0
```

Comment:
Not sure if this is correct, @zdtsw ?

Reply:
@lburgazzoli @zdtsw I'll revert it to the value on main. This seems to be related to `VERSION ?= 2.33.0` on the `Makefile`.

Reply:
Reverted to 3.0.0

Reply:
yes. let me fix on the main first

Reply:
Resolved conflicts.